doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -583,7 +583,16 @@ Other samples
   * Added:
 
     * Support for the nRF54L05 and nRF54L10 SoCs (emulated on nRF54L15 DK).
-    * FLPR core support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target.
+    * FLPR core support for the :ref:`zephyr:nrf54l15dk_nrf54l15` and :ref:`zephyr:nrf54h20dk_nrf54h20` board targets.
+
+  * Removed the following compiler options that were set in the :kconfig:option:`CONFIG_COMPILER_OPT` Kconfig option:
+
+    * ``-fno-pie``
+    * ``-fno-pic``
+    * ``-ffunction-sections``
+    * ``-fdata-sections``
+
+    These options are enabled by default in Zephyr and do not need to be set with the dedicated Kconfig option.
```

Comment:
```suggestion
    These options are enabled by default in Zephyr.
    You do not need to set them with a dedicated Kconfig option.
```

Reply:
discussed in the other comm channel, decide to keep it as it is